Chatkit/Redux/Store/State.swift

```diff
@@ -0,0 +1,34 @@
+
+enum Internal {}
```

Comment:
Isn't this a code smell that this fake namespace is not required and regular Swift access controls should be sufficient?

Reply:
I need a namespace otherwise `Room` and `User` etc will clash with their `public` counterparts.  

At any rate this entire class is just a WIP and I expect it to be entirely rewritten.  This will become your baby since you are working on the `State` and `Reducers`.  You are free to arrange and name this as you wish. 

If you'd like me to change it to something else for this PR then make a suggestion and I'll do so.

Reply:
Let's leave `Internal` then.